suite-native/atoms/src/TransactionItem/TransactionItemWrapper.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { ReactNode } from 'react';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { VStack } from '../VStack';
+
+type TransactionItemWrapperProps = {
+    children: ReactNode;
+};
+
+const transactionItemWrapperStyle = prepareNativeStyle(utils => ({
+    paddingRight: utils.spacings.medium,
+    paddingLeft: 12,
+    borderRadius: 12,
+}));
+
+export const TransactionItemWrapper = ({ children }: TransactionItemWrapperProps) => {
```

Comment:
I am not sure if we are going to need this component at all, I think we will something like more universal `Card` component because it just white box with rounded corners which is used lot of times in designs.

Reply:
Ok, let's do that in https://github.com/trezor/trezor-suite/issues/5711

Reply:
https://github.com/trezor/trezor-suite/pull/5783/commits/eebfdd500df1a8017caed50b05949391633cbb86